content/guides/hosting/hosting-options/_index.md

```diff
@@ -6,4 +6,29 @@ menu:
 title: Deployment options
 weight: 1
 ---
+This section describes the different ways can you can deploy W&B.
 
+## W&B Multi-tenant Cloud
+[W&B Multi-tenant Cloud]({{< relref "saas_cloud.md" >}}) is fully managed by W&B, including upgrades, maintenance, platform security, and capacity planning. Multi-tenant Cloud is deployed in W&B's Google Cloud Platform (GCP) account in [GPC's North America regions](https://cloud.google.com/compute/docs/regions-zones).
+
+[Learn more about W&B Multi-tenant Cloud]({{< relref "saas_cloud.md" >}}) or [get started for free](https://app.wandb.ai/login?signup=true).
+
+## W&B Dedicated Cloud
```

Comment:
Might want to mention BYOB here. That was(is?) a concern for customers.

Reply:
I believe BYOB is supported for multi-tenant too: https://docs.wandb.ai/guides/hosting/data-security/secure-storage-connector/

Reply:
PTAL at the suggested edits before I apply them.